Assets/MXR.SDK/Runtime/Types/StatusTypes.cs

```diff
@@ -122,6 +126,47 @@ public class Controller {
         public string version = string.Empty;
     }
 
+    /// <summary>
+    /// Enables tracking and reporting of network errors and connectivity events across different SSIDs on a device.
+    /// </summary>
+    [System.Serializable]
+    public class NetworkErrorCodeFrequency {
+        public List<NetworkErrorStatistics>? networkErrorStatistics = new List<NetworkErrorStatistics>();
+        public int? connectivityFlippedCount;
+
+        public NetworkErrorCodeFrequency() { }
```

Comment:
Same here, why do we need these constructors if we're deserializing from json string?